src/kata6.joinNames.js

```diff
@@ -1,3 +1,15 @@
-const joinNames = namesObj => {};
-
+const joinNames = namesObj => {;
+ let newString = '';
+ const  arrayLength = namesObj.length;
+ for (let i = 0; i < arrayLength; i++) {
```

Comment:
Nice use of a for loop here.

Reply:
Extra challenge - can you refactor this using an iterator method rather than a for loop?